graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -263,11 +263,19 @@ public SearchResult search(SearchesConfig config) {
         return new SearchResult(r.getHits(), indices, config.query(), request.source(), r.getTook());
     }
 
-    public TermsResult terms(String field, int size, String query, String filter, TimeRange range) {
+    public TermsResult terms(String field, int size, String query, String filter, TimeRange range, String sorting) {
```

Comment:
Please don't use a string for `sorting` here but [`Sorting.Direction`](https://github.com/Graylog2/graylog2-server/blob/2.2.1/graylog2-server/src/main/java/org/graylog2/indexer/searches/Sorting.java#L30-L33) instead.